understanding/pronunciation.html

```diff
@@ -55,154 +53,116 @@
-                  each with its own pronunciation. The meaning of such words or characters can usually
-                  be determined from the context of the sentence. However, for more complex or ambiguous
-                  sentences, or for some languages, the meaning of the word cannot be easily determined
-                  or determined at all without knowing the pronunciation. When the sentence is read
-                  aloud and the screen reader reads the word using the wrong pronunciation, it can be
-                  even more difficult to understand than when read visually. When words are ambiguous
-                  or indeterminate unless the pronunciation is known, then providing some means of determining
-                  the pronunciation is needed.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、意味が発音に依存する場合のコンテンツを理解するために全盲の利用者、ロービジョンの利用者、及び読字障害のある利用者を支援することである。多くの場合、単語や文字は異なる意味を持ち、それぞれ独自の発音をする。そのような単語又は文字の意味は、文の前後の文脈から通常は決定される。しかし、より複雑又は曖昧な文章、いくつかの言語においては、発音を知らないと、その単語の意味を容易に決定できない又は全く決定できないことがある。文を音読やスクリーンリーダーによって間違った発音で単語を読むと、視覚的に読むよりも理解するのがさらに難しくなる可能性がある。発音が分からない限り、単語が曖昧又ははっきりしない場合は、発音を判断する手段を提供する必要がある。
                   
                </p>
                <p>
-                  	For example, in the English language heteronyms are words that are spelled the same
-                  but have different pronunciations and meanings, such as the words desert (abandon)
-                  and desert (arid region). If the proper pronunciation can be determined from the context
-                  of the sentence, then nothing is required.  If it cannot then some mechanism for determining
-                  the proper pronunciation would be required. Additionally, in some languages certain
-                  characters can be pronounced in different ways. In Japanese, for example, there are
-                  characters like Han characters(Kanji) that have multiple pronunciations. Screen readers
-                  may speak the characters incorrectly without the information on pronunciation. When
-                  read incorrectly, the content will not make sense to users.
+                  	たとえば、英語の同形異音異義語の "desert" (見捨てる) と"desert" (砂漠) のように、スペルは同じでも発音と意味が異なる単語がある。適切な発音を文の文脈から決定できる場合は、何も必要とされない。それができない場合は、適切な発音を決定するための何らかの仕組みが必要になる。さらに、一部の言語は、特定の文字が異なる方法で発音されることがある。例えば、日本語では、複数の発音を持つ漢字のような文字がある。スクリーンリーダーは、発音に関する情報がないと、文字を間違って話すことがある。間違って読むと、コンテンツは利用者にとって意味をなさなくなるだろう。
                   
                </p>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
-               <p>This Success Criterion may help people who: 
+               <h2>利点</h2>
+               <p>この達成基準は、次のような利用者にメリットがある:
                   
                   
                </p>
                <ul>
                   
                   
-                  <li>have difficulty decoding words</li>
+                  <li>単語を復号化しづらい。</li>
```

Comment:
https://github.com/waic/wcag22/pull/417#discussion_r2088551964 と揃えると:

```suggestion
                  <li>単語を読み解くことが困難である。</li>
```